features/unsupported/tests/mbed/can_api/main.cpp

```diff
@@ -0,0 +1,85 @@
+#include "mbed.h"
+#include "test_env.h"
+
+#if !DEVICE_CAN
+#error [NOT_SUPPORTED] CAN not supported
+#endif
+
+#if defined(TARGET_LPC1549)
+#define CAN_RD D9
+#define CAN_TD D8
+#elif defined(TARGET_LPC1768) || defined(TARGET_LPC4088)
+#define CAN_RD p9
+#define CAN_TD p10
+#elif defined(TARGET_B96B_F446VE)
+#define CAN_RD PD_0
+#define CAN_TD PD_1
+#elif defined(TARGET_VK_RZ_A1H)
+#define CAN_RD P5_9
+#define CAN_TD P5_10
+#elif defined(TARGET_NUCLEO_F042K6) || \
+      defined(TARGET_NUCLEO_F072RB) || \
+      defined(TARGET_NUCLEO_F091RC) || \
+      defined(TARGET_NUCLEO_F302R8) || \
+      defined(TARGET_NUCLEO_F303RE) || \
+      defined(TARGET_NUCLEO_F303K8) || \
+      defined(TARGET_NUCLEO_F334R8) || \
+      defined(TARGET_NUCLEO_F412ZG) || \
+      defined(TARGET_DISCO_F429ZI)  || \
+      defined(TARGET_NUCLEO_F446RE) || \
+      defined(TARGET_NUCLEO_F746ZG) || \
+      defined(TARGET_NUCLEO_L432KC) || \
+      defined(TARGET_DISCO_L476VG)  || \
+      defined(TARGET_NUCLEO_L476RG)
+#define CAN_RD PA_11
+#define CAN_TD PA_12
+#elif defined(TARGET_NUCLEO_F103RB) || \
+      defined(TARGET_NUCLEO_F207ZG) || \
+      defined(TARGET_DISCO_F303VC)  || \
+      defined(TARGET_NUCLEO_F303ZE) || \
+      defined(TARGET_NUCLEO_F412ZG) || \
+      defined(TARGET_NUCLEO_F429ZI) || \
+      defined(TARGET_NUCLEO_F439ZI) || \
+      defined(TARGET_NUCLEO_F446ZE) || \
+      defined(TARGET_DISCO_F469NI)  || \
+      defined(TARGET_DISCO_F746NG)  || \
+      defined(TARGET_NUCLEO_F756ZG) || \
+      defined(TARGET_NUCLEO_F767ZI) || \
+      defined(TARGET_DISCO_F769NI)  || \
+      defined(TARGET_NUCLEO_L486RG)
+#define CAN_RD PB_8
+#define CAN_TD PB_9
+#endif
+
+int result = true;
+
+void Check_CAN_Frequency(int CanFrequency)
+{
+    printf("Init CAN at %d Hz\n", CanFrequency);
+
+    CAN can_object(CAN_RD, CAN_TD, CanFrequency);
+
+#if !defined(TARGET_VK_RZ_A1H)
```

Comment:
It is inherited from test/mbed/can_loopback/main.cpp (ie MBED_A27)
We don't have the material to test it, we did not want to take the responsibility of removing a code we don't know.
Cheers